### PR TITLE
Mac ci

### PR DESCRIPTION
**Pull request by bam241**
first attempt to setup MACOS CI

### CODE REVIEW

**Comment by gonuke** [comment]
Maybe start with the build strategy in the condo-forge Moab-feedstock? (Just a thought)

File: .github/workflows/mac_build_test.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      - develop
+      - master
+  release:
+    types: # This configuration does not affect the page_build event above
+      - created
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
```
Comment (gonuke) [inline .github/workflows/mac_build_test.yml]:
```suggestion
  BuildTest:
```

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks for sorting this out @bam241 - a little change for consitency

**Comment by gonuke** [comment]
Thanks for the quick work here @bam241. I'm merging this first because the rebase will probably be simpler this way around

**Comment by pshriwise** [comment]
Thanks for tracking down the cause of the problem here @bam241! Sorry for not including that header initially.

**Pull request merged**